style-guide/styles/wds-mixins/_button-group-theming.scss

```diff
@@ -0,0 +1,27 @@
+@mixin button-group-theming($color-border) {
+	> .wds-button {
+		&:not(.wds-is-secondary) {
+			&:not(:last-child) {
+				border-right: 1px solid $color-border;
```

Comment:
@rybmat I'm not sure if we should treat `1px solid` as theming, it doesn't change per theme. IMO it would be cleaner if we had only the colors set here.